templates/ovncontroller/bin/start-vswitchd.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
(no action required) I know it's also `sh` (and not `bash`) in other scripts here, but I wonder if we should just use `bash` here - I for one am not very knowledgeable to confirm that this script doesn't use any bash-isms.